### ISSUE TITLE
Design Kick-off meeting

### ISSUE THREAD

**Issue by AlexKhotian** [AlexKhotian/DiCh#6]
Here we want to:

- [ ] Create backbone map;
- [ ] Create road map for future milestones;
- [ ] Define architecture questions;
- [ ] Define user stories.

Every team member is allowed to add new points.

[Survey for possible dates](https://doodle.com/poll/cy3h7gbr3pvk7tb7)

#5

**Comment by AlexKhotian** [comment]
@sergey546 @anastasiasuslova @ZoiaRapchins seems like 26th and 27th. Now we have to find a location? Stuttgart or Mannheim?

**Comment by anastasiasuslova** [comment]
You mean 27th and 28th? 
I vote for Weinheim! (:

**Comment by AlexKhotian** [comment]
@anastasiasuslova 
Yeah... perhaps it will be hard for us to manage one more trip at 27th of May. We are surrounded by trips XD. So you may conceder to come to us or we can shift it to second half of June?

**Comment by AlexKhotian** [comment]
@anastasiasuslova @sergey546  @ZoiaRapchins 
Do we want to continue(start) this project?

**Comment by sergey546** [comment]
I want
